bin/setup

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+set -euo pipefail
+IFS=$'\n\t'
+set -vx
+
+for path in gemfiles/Gemfile_*.rb; do
+  BUNDLE_GEMFILE="${path}" bundle install
+done
```

Comment:
Following the convention of the template prepared by recent Bundler's `bundle gem` command, I decided o put the shell scripts to be used for development such as `bin/setup` and `bin/test` under the `bin` directory.